pkg/manifests/manifests.go

```diff
@@ -261,6 +318,77 @@ func (f *Factory) RouterServiceCloud(cr *ingressv1alpha1.ClusterIngress) (*corev
 	return s, nil
 }
 
+func (f *Factory) MetricsServiceMonitor(ci *ingressv1alpha1.ClusterIngress) (*monv1.ServiceMonitor, error) {
+	sm, err := NewServiceMonitor(MustAssetReader(MetricsServiceMonitor))
+	if err != nil {
+		return nil, err
+	}
+	sm.Name = "router-" + ci.Name
+
+	for i := range sm.Spec.Endpoints {
+		if sm.Spec.Endpoints[i].Path == "/metrics" {
+			sm.Spec.Endpoints[i].Port = string(MetricsListenAddressPort)
+			sm.Spec.Endpoints[i].TLSConfig.ServerName = fmt.Sprintf("%s.%s.svc", sm.Name, sm.Namespace)
+			sm.Spec.Endpoints[i].BasicAuth = &monv1.BasicAuth{
```

Comment:
As previously discussed, there shouldn't be any basic auth for the metrics endpoint exposed by the router controller